README.md

```diff
@@ -52,3 +62,63 @@ docker run
+  + *right_ascension*: (optional) the right ascension of the observation target. This is not needed if you are performing an observation in a stationary position and the telescope will not slew. String.
+  + *declination*: (optional) the declination of the observation target. This is not needed if you are performing an observation in a stationary position and the telescope will not slew. String.
+  + *beamwidth:* (optional) beamwidth of the telescope. This is optional and will assign a default value of 3 if left unassigned. float.
+  + *height*: (optional) height of the telescope. float.
+  + *azimuth*: (optional) azimuth of the telescope. Only used if performing an observation in a stationary position so not often needed. float.
+  + *elevation:* (optional) elevation (altitude) of the telescope. Only used if performing an observation in a stationary position so not often needed. float.
++ **Satellite**: the satellite data class stores all of the TLE information for each satellite, which is loaded from a TLE file using the class method from_tle_file() and can be converted to a Skyfield API object EarthSatellite using the to_rhodesmill() method. It also stores all the frequency information for each satellite.
+  + *name:* name of satellite. string.
+  + *tle_information:* stores TLE information. TleInformation is another custom object to store TLE data and can be found in ~/custom_dataclasses/satellite/tle_information.py
+  + *frequency:* list of type FrequencyRange. FrequencyRange is a custom dataclass that stores a center frequency and bandwidth.
+  + *to_rhodesmill():* class method to convert a Satellite object into a Rhodemill-Skyfield EarthSatellite object for use with the Skyfield API
+  + *from_tle_file()*: class method to load Satellite from provided TLE file. Returns a list of type Satellite.
++ **Reservation:** stores the Facility, as well as some additional reservation-specific information, such as reservation start and end times.
+  + *facility*: Facility (see above)
+  + *time*: TimeWindow that contains the start and end time of the ideal reservation. Type TimeWindow (see above).
+  + *frequency*: frequency of the requested observation. Type FrequencyRange.
++ **OverheadWindow:** designed to store the time windows that a given satellite is overhead and includes the Satellite object, as well as a TimeWindow object that contains the interference start and end times.
+  + *satellite*: the satellite that is overhead during the time window. Type Satellite.
+  + *overhead_time*: the time the satellite enters and exits view. TimeWindow.
++ **ObservationPath:** represents the telescope's position as a function of time by storing an altitude, azimuth, and timestamp
+  + *altitude*: altitude of the telescope. Angle (type from Astropy).
+  + *azimuth*: azimuth of the telescope. Angle (from Astropy).
+  + *time*: time at which the telescope is at the stored altitude and azimuth. datetime.
+### Functional Modules
```

Comment:
Seems like these should be [docstrings](https://peps.python.org/pep-0257/).

Reply:
Moved all of these into their respective files and deleted from README